server/src/internalClusterTest/java/org/opensearch/snapshots/CloneSnapshotIT.java

```diff
@@ -338,7 +338,7 @@ public void testBackToBackClonesForIndexNotInCluster() throws Exception {
         final String targetSnapshot1 = "target-snapshot";
         blockMasterOnShardClone(repoName);
```

Comment:
`blockMasterOnShardClone` -> `blockClusterManagerOnShardClone`?

Reply:
Changed the method name in the new commit https://github.com/opensearch-project/OpenSearch/commit/87d07e5b81ed67d4f72842ef335ae627839e4954. Thank you! 👍